docs/Documentation/tutorial.md

```diff
@@ -43,191 +45,6 @@ VIC model), such as:
      * Net upward sensible heat
      * ... ...
 *  ***Routed streamflow at specified stream locations***
-     * (can be any locations within the stream network, and the resolution is the same as grid cell, i.e., stream locations are specified by indicating the grid cell in which it falls) calculated by the routing model, daily or subdaily time step.
```

Comment:
If you do have to include this, then at least explicitly say what you said (i.e., these are generated by VIC disaggregator)